public/javascript/index.js

```diff
@@ -1,23 +1,154 @@
-const charactersAPI = new APIHandler('http://localhost:8000');
+const charactersAPI = new APIHandler('https://minions-api.herokuapp.com');
 
 window.addEventListener('load', () => {
-  document.getElementById('fetch-all').addEventListener('click', function (event) {
+  document
+    .getElementById('fetch-all')
+    .addEventListener('click', function (event) {
+      charactersAPI
+        .getFullList()
+        .then(response => {
+          let text = '';
+          response.data.reverse().forEach(
+            minion =>
+              (text += `<div class="character-info">
+       <div class="name"> ID: ${minion.id}</div>
+        <div class="name"> Name: ${minion.name}</div>
+        <div class="occupation">Occupation: ${minion.occupation}</div>
+        <div class="cartoon">Cartoon: ${minion.cartoon}</div>
+        <div class="weapon">Weapon: ${minion.cartoon}</div>
+      </div>`)
+          );
+          document.querySelector('.characters-container').innerHTML = text;
+        })
+        .catch(err => console.log('ERROR', err));
+    });
 
-  });
+  document
+    .getElementById('fetch-one')
+    .addEventListener('click', function (event) {
+      const valueId = document.querySelector('.character-id').value;
 
-  document.getElementById('fetch-one').addEventListener('click', function (event) {
+      charactersAPI
+        .getOneRegister(valueId)
+        .then(response => {
+          let text = '';
+          const minion = response.data;
+          text += `<div class="character-info">
+           <div class="name"> ID: ${minion.id}</div>
+            <div class="name"> Name: ${minion.name}</div>
+            <div class="occupation">Occupation: ${minion.occupation}</div>
+            <div class="cartoon">Cartoon: ${minion.cartoon}</div>
+            <div class="weapon">Weapon: ${minion.cartoon}</div>
+          </div>`;
+          document.querySelector('.characters-container').innerHTML = text;
+        })
+        .catch(err => {
+          console.log('ERROR', err);
+          document.querySelector('#fetch-one').classList = 'err';
```

Comment:
Si quieres sustituir todas las clases por una en concreto es mejor usar la propiedad classname.

```suggestion
          document.querySelector('#fetch-one').className = 'err';
```

ClassList es para trabajo más "fino", quitar y poner clases individuales, remplazarlas etc.